docs/dev/contributing.md

```diff
@@ -55,6 +56,23 @@ Please rebase your changes on `master` and squash your commits whenever possible
 it keeps history cleaner and it's easier to revert things. It also makes developers
 happier!
 
+### Reno
```

Comment:
Can you add a very quick note on how to install reno in a dev env? (I guess mentioning `pip install reno` is enough)

Reply:
good call !